+ source ../common/common_lib.sh
++ source /root/mugen/libs/locallibs/common_lib.sh
+++ python3 --version
+++ '[' 0 -eq 0 ']'
+++ source /root/mugen/libs/locallibs/common_lib_python.sh
++ main
+++ type -t post_test
++ '[' -n '' ']'
++ trap POST_TEST_DEFAULT EXIT INT HUP TERM
++ rpm -qa
++ grep expect
+++ type -t config_params
++ '[' -n '' ']'
+++ type -t pre_test
++ '[' -n '' ']'
+++ type -t run_test
++ '[' -n '' ']'
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n function ']'
+ pre_test
+ LOG_INFO 'Start to prepare the test environment.'
+ message='Start to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to prepare the test environment.'
Wed Apr  5 08:47:15 2023 - INFO  - Start to prepare the test environment.
+ Pre_Test
+ DNF_INSTALL ansible
+ pkgs=ansible
+ node=1
+ '[' -z '' ']'
+ tmpfile=
++ python3 /root/mugen/libs/locallibs/rpm_manage.py install --pkgs ansible --node 1 --tempfile ''
+ tmpfile2=/tmp/tmpt5utulx4
+ '[' -z '' ']'
+ tmpfile=/tmp/tmpt5utulx4
+ sed -i '1i\${NODE2_IPV4}' /etc/ansible/hosts
+ expect
spawn ssh-keygen
Generating public/private rsa key pair.

Enter file in which to save the key (/root/.ssh/id_rsa): 
Created directory '/root/.ssh'.
Enter passphrase (empty for no passphrase): 
Enter same passphrase again: + expect
spawn ssh-copy-id root@
/usr/bin/ssh-copy-id: ERROR: No identities found
send: spawn id exp3 not open
    while executing
"send "yes\n""
+ echo ''
+ LOG_INFO 'End to prepare the test environment.'
+ message='End to prepare the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to prepare the test environment.'
Wed Apr  5 08:50:50 2023 - INFO  - End to prepare the test environment.
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Wed Apr  5 08:50:51 2023 - INFO  - Start to run test.
+ ansible-inventory --help
+ grep Usage:
+ CHECK_RESULT 1 0 0 'Check ansible-inventory --help failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ansible-inventory --help failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ansible-inventory --help failed.'
+ LOG_ERROR 'Check ansible-inventory --help failed.'
+ message='Check ansible-inventory --help failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check ansible-inventory --help failed.'
Wed Apr  5 08:51:04 2023 - ERROR - Check ansible-inventory --help failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_ansible_04.sh line 28'
+ message='oe_test_ansible_04.sh line 28'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_ansible_04.sh line 28'
Wed Apr  5 08:51:05 2023 - ERROR - oe_test_ansible_04.sh line 28
+ '[' 0 -eq 1 ']'
+ return 0
+ ansible-inventory --version
+ grep ansible-inventory
ansible-inventory 2.9.27
  executable location = /usr/bin/ansible-inventory
+ CHECK_RESULT 0 0 0 'Check ansible-inventory --version failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-inventory --version failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ expect
spawn ansible-console
exit
+ CHECK_RESULT 0 0 0 'Check ansible-console failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-console failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ expect
spawn ansible-console --ask-vault-pass

exit
+ CHECK_RESULT 0 0 0 'Check ansible-console --ask-vault-pass failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-console --ask-vault-pass failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ expect
spawn ansible-console --vault-password-file=/tmp/pass
exit
+ CHECK_RESULT 0 0 0 'Check ansible-console --vault-password failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-console --vault-password failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ ansible-playbook --help
+ grep Usage:
+ CHECK_RESULT 1 0 0 'Check ansible-playbook --help failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ansible-playbook --help failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ansible-playbook --help failed.'
+ LOG_ERROR 'Check ansible-playbook --help failed.'
+ message='Check ansible-playbook --help failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check ansible-playbook --help failed.'
Wed Apr  5 08:52:14 2023 - ERROR - Check ansible-playbook --help failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_ansible_04.sh line 55'
+ message='oe_test_ansible_04.sh line 55'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_ansible_04.sh line 55'
Wed Apr  5 08:52:16 2023 - ERROR - oe_test_ansible_04.sh line 55
+ '[' 0 -eq 1 ']'
+ return 0
+ ansible-playbook --version
+ grep ansible-playbook
ansible-playbook 2.9.27
  executable location = /usr/bin/ansible-playbook
+ CHECK_RESULT 0 0 0 'Check ansible-playbook --version failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-playbook --version failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep Usage:
+ ansible-pull --help
+ CHECK_RESULT 1 0 0 'Check ansible-pull --help failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ansible-pull --help failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ansible-pull --help failed.'
+ LOG_ERROR 'Check ansible-pull --help failed.'
+ message='Check ansible-pull --help failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check ansible-pull --help failed.'
Wed Apr  5 08:52:44 2023 - ERROR - Check ansible-pull --help failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_ansible_04.sh line 59'
+ message='oe_test_ansible_04.sh line 59'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_ansible_04.sh line 59'
Wed Apr  5 08:52:45 2023 - ERROR - oe_test_ansible_04.sh line 59
+ '[' 0 -eq 1 ']'
+ return 0
+ ansible-pull --version
+ grep ansible-pull
ansible-pull 2.9.27
  executable location = /usr/bin/ansible-pull
+ CHECK_RESULT 0 0 0 'Check ansible-pull --version failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-pull --version failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ grep Usage:
+ ansible-vault --help
+ CHECK_RESULT 1 0 0 'Check ansible-vault --help failed.'
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log='Check ansible-vault --help failed.'
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n 'Check ansible-vault --help failed.'
+ LOG_ERROR 'Check ansible-vault --help failed.'
+ message='Check ansible-vault --help failed.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Check ansible-vault --help failed.'
Wed Apr  5 08:53:14 2023 - ERROR - Check ansible-vault --help failed.
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_ansible_04.sh line 63'
+ message='oe_test_ansible_04.sh line 63'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_ansible_04.sh line 63'
Wed Apr  5 08:53:15 2023 - ERROR - oe_test_ansible_04.sh line 63
+ '[' 0 -eq 1 ']'
+ return 0
+ ansible-vault --version
+ grep ansible-vault
ansible-vault 2.9.27
  executable location = /usr/bin/ansible-vault
+ CHECK_RESULT 0 0 0 'Check ansible-vault --version failed.'
+ actual_result=0
+ expect_result=0
+ mode=0
+ error_log='Check ansible-vault --version failed.'
+ exit_mode=0
+ '[' -z 0 ']'
+ '[' 0 -eq 0 ']'
+ test 0x '!=' 0x
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Wed Apr  5 08:53:31 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 4
+ test 4 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Wed Apr  5 08:53:32 2023 - ERROR - failed to execute the case.
+ exit 4
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Wed Apr  5 08:53:33 2023 - INFO  - Start to restore the test environment.
+ Post_Test
+ rm -rf /etc/ansible/hosts
+ DNF_REMOVE
+ node=1
+ pkg_list=
+ mode=0
+ [[ -z /tmp/tmpt5utulx4 ]]
+ '[' 0 -ne 0 ']'
+ '[' 1 == 0 ']'
+ python3 /root/mugen/libs/locallibs/rpm_manage.py remove --node 1 --pkgs '' --tempfile /tmp/tmpt5utulx4
+ '[' 0 -ne 0 ']'
+ rm -rf /root/.ssh
+ rm -rf /tmp/pass
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Wed Apr  5 08:54:49 2023 - INFO  - End to restore the test environment.
